src/app/caesar-portal/common/models/group.ts

```diff
@@ -3,15 +3,12 @@ import { User }     from './user';
 import { Location } from './location';
 
 export class Group {
-  stage: string;
-  location: Location;
-  teachers: User[];
-
   constructor(public groupId: number = 0, 
               public name: string = '', 
               public startDate: string = null, 
               public finishDate: string = null, 
-              public experts: string[] = null) { }
+              public experts: string[] = null,
+              public _links: string[] = null) { }
```

Comment:
why name _links? if you wanted it to be private than make it private without any _ in the name